data/human/hails.txt

```diff
@@ -1982,24 +1973,120 @@ phrase "friendly civilian"
-		"the Quarg will arrest you if they catch a whiff of any of their equipment on your ship."
-		"the Quarg imprison anyone they catch carrying their outfits."
-		"nine out of ten pirates die within their first year."
-		"Republic Intelligence has their eyes everywhere."
-		"the law will always catch up to those who try to bypass it."
+		"."
+	replace
+		"It's said that the Quarg make it pretty clear that they" "It's said that the Quarg"
+
+phrase "friendly civilian"
+	word
+		"What is with zero-g cricket?"
+		"I don't get that zero-g cricket they play in the Core."
+		"How are people into zero-g cricket?"
+		"How do people stay awake watching that zero-g cricket?"
+	word
+		" I mean, "
+		" Look, "
+		" Like, "
+	word
+		"all they're doing is"
+		"half the time they're just"
+		"they spend a week"
+		"you can't call a sport a sport if you're mostly "
+		"all you're watching is a bunch of guys"
+	word
+		" floating around "
+		" floating aimlessly "
+		" bouncing around a space station "
+	word
+		"trying to catch a ball"
+		"hitting a ball around a space station"
+		"while two of them go back and forth a bunch of times"
+		"and boosting about twenty meters back and forth every now and again"
+		"trying to hit a couple sticks off a few other sticks with a ball"
+	word
+		"" 5
+		". I just don't get it, man"
+		". It makes no sense"
+		". Don't get me started on the rules"
+		". Innings, outings, overs, it's ridiculous"
+		". Who's in and who's out? They never tell you"
+		". It's too bloody convoluted, the way people go in and out all the time"
+		". It sounds like a sport only aliens would play"
+		". Couldn't they speed it up a bit?"
+		". How do they make a game out of that?"
+		". And all they're playing for is a bunch of old burnt wood"
+		`. And they're always up in arms about whether "underarm bowling" is legal`
+		". Zero-g lacrosse makes a lot more sense"
+	word
+		"!"
+	replace
+		"?!" "?"
+		"around a space station hitting a ball around a space station" "around hitting a ball around a space station"
+
+phrase "friendly civilian"
+	word
+		"I'm getting into zero-g cricket"
+		"I love watching zero-g cricket"
+		"You don't get a sport better than zero-g cricket"
+	word
+		". That adrenaline you get when the bowler boosts down the sphere to throw the ball"
+		". The exhilaration you feel when the batsman hits a perfect six"
+		". The way bowlers can control the ball to fool the batsman"
+		". That feeling you get when a fielder catches the batsman out with a well-timed boost"
+	word
+		" and the way the crowd roars as it happens"
+		", the crowd going wild"
+		", and the release you feel as you see the ball getting knocked right out of the sphere"
+		", how each test is full of tension and fortunes that are decided by each bowl and hit"
+		"" 6
+	word
+		"; there's nothing like it in the galaxy."
+		"; I don't get the hate for it."
+		"... There are some truly phenomenal players out there."
```

Comment:
```suggestion
		"...There are some truly phenomenal players out there."
```

Reply:
I don't usually see an elipsis without a space after it, but apparently this is a thing.

Reply:
Technically this isn't an ellipsis, it's 'suspension points'. An ellipsis indicates that something is missing and suspension points show hesitation or a pause.

In this specific case I would keep the space but drop the capitalisation, as the sentence is running on from whichever previous phrase was displayed.

```suggestion
		"... there are some truly phenomenal players out there."
```